doc/introduction/introducing_zephyr.rst

```diff
@@ -29,8 +30,44 @@ as described in :ref:`Zephyr_Licensing`.
 Distinguishing Features
 ***********************
 
-The Zephyr kernel offers a number of features that distinguish it from other
-small-footprint OSes:
+Zephyr offers a large and evergrowing number of features including:
+
+**Extensive suite of Kernel services**
+   Zephyr offers a number of familiar services for development:
+
+   * *Multi-threading Services* for cooperative, priority-based,
+     non-preemptive, and preemptive threads with optional round robin
+     time-slicing. Includes POSIX pthreads compatible API support.
+
+   * *Interrupt Services* for compile-time registration of interrupt handlers.
+
+   * *Memory Allocation Services* for dynamic allocation and freeing of
+     fixed-size or variable-size memory blocks.
+
+   * *Inter-thread Synchronization Services* for binary semaphores,
+     counting semaphores, and mutex semaphores.
+
+   * *Inter-thread Data Passing Services* for basic message queues, enhanced
+     message queues, and byte streams.
+
+   * *Power Management Services* such as tickless idle and an advanced idling
+     infrastructure.
+
+**Multiple Scheduling Algorithms**
+
```

Comment:
My bad... delete this blank line (57)  to have the dictionary list display correctly.